packages/plugin-auto-tracking-browser/src/helpers.ts

```diff
@@ -0,0 +1,53 @@
+const SENTITIVE_TAGS = ['input', 'select', 'textarea'];
+
+export const isNonSensitiveString = (text: string | null) => {
+  if (text == null) {
+    return false;
+  }
```

Comment:
Will email also be the sensitive info for some customers?

Reply:
I think email is probably fine, not sensitive but PII, as the normal instrumentation would require user id/email to send events.